build.cmd

```diff
@@ -204,6 +204,19 @@ call                                 "%__VSToolsRoot%\VsDevCmd.bat"
 
 REM =========================================================================================
 REM ===
+REM === Restore optimization profile data
+REM ===
+REM =========================================================================================
+
+echo %__MsgPrefix%Restoring the OptimizationData Package
```

Comment:
@dpodder, along the same lines, do we want nopgo to be the default or do we want pgo to be the default?  That would likely affect what change you make here.